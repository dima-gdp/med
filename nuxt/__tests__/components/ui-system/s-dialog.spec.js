import { shallowMount } from '@vue/test-utils'
import SDialog from '~/components/ui-system/s-dialog'
// todo: протестировать фичи clickOutside, закрытия по esc, отображения про разных props
describe('ui-system/s-dialog', () => {
  const dialogFactory = (propsData = {}) => {
    return shallowMount(SDialog, {
      props: {
        ...propsData,
      },
    })
  }

  it('ui-dialog отображается', () => {
    const wrapper = dialogFactory({ value: true })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
