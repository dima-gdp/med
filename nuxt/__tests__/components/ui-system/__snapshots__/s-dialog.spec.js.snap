// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-system/s-dialog ui-dialog отображается 1`] = `
<transition-stub name="modal">
  <!---->
</transition-stub>
`;
